README.dev.md

```diff
@@ -50,6 +50,10 @@ You can set the cloud provider regions for the image to be copied over by editin
 
  After saving the `Makefile` the modified values are applied to all subsequent image burns.
 
+Note: If you experience a failure during the SSH connection step, you might need to adjust the subnet_id and vpc_id values
+within the packer.json file.  By default the transient EC2 instance will be created in the same VPC and Subnet as the 
```

Comment:
These can be passed in in env variables now e.g.:
export SUBNET_ID=subnet-adfasdfdsfsdf
export VPC_ID=vpc-sdfsdfsdfsdfsdf